modules/ROOT/pages/type-definitions/types/index.adoc

```diff
@@ -29,27 +28,27 @@ The Neo4j GraphQL Library provides the following types:
 | The `BigInt` scalar type represents a signed 64-bit numeric non-fractional value.
 
 | xref:/type-definitions/types/temporal.adoc[`Date`]
-| The `Date` scalar type is a ISO 8601 date mapping to the Neo4j `DATE` type.
+| The `Date` temporal type is a ISO 8601 date mapping to the Neo4j `DATE` type.
```

Comment:
An argument could be made for calling these "temporal scalar" types, as you could argue that all temporal types are a subset of scalar types.

Perhaps that's needlessly pedantic, however?

Reply:
hm... you're right.
we currently differentiate between scalar and temporal types implicitly because we have dedicated pages for both. should we instead include the temporal types in the scalar types page?

Reply:
Having a separate page for temporal types is probably the right choice. As a reader, I reckon I'd be able to find them more easily if there was a separate page.

The change I'm thinking of would be to name them `temporal scalar type` instead of `temporal type` on this index page.